File: service/src/main/java/ru/diasoft/micro/dofacade/mock/LoanReq.java
package ru.diasoft.micro.dofacade.mock;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class LoanReq {
    @JsonProperty("NAME")
    String name;
    @JsonProperty("MOBILE_NUMBER")
    String mobileNumber;
    @JsonProperty("IC_NUMBER")
    String icNumber;
    @JsonProperty("LEAD_NUMBER")
    String leadNumber;
    @JsonProperty("LEAD_SOURCE")
    String leadSource;
    @JsonProperty("COMMENT")
    String comment;
}

File: src/main/java/ru/diasoft/micro/dofacade/json/application/LoanReq.kt
package ru.diasoft.micro.dofacade.json.application

import com.fasterxml.jackson.annotation.JsonProperty

data class LoanReq(
    @JsonProperty("NAME")
    val name: String,
    @JsonProperty("MOBILE_NUMBER")
    val mobileNumber: String,
    @JsonProperty("IC_NUMBER")
    val icNumber: String,
    @JsonProperty("LEAD_NUMBER")
    val leadNumber: String,
    @JsonProperty("LEAD_SOURCE")
    val leadSource: String,
    @JsonProperty("COMMENT")
    val comment: String
)
